db/replication-alarms.yml

```diff
@@ -0,0 +1,97 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  Creates alarms related to ssh tunnel remote server -> rds db replication
+Parameters:
+  OpsWarnSnsTopic:
+    Default: ''
+    Description: ops warn SNS Topic arn
+    Type: String
+  EnvironmentType:
+    Type: String
+    AllowedValues:
+      - staging
+      - production
+  DatabaseName:
+    Default: 'mediajoint_production'
+    Description: Name of the database
+    Type: String
+  TunnelInstanceId:
+    Default: ''
+    Description: Instance id (i.e. i-something) of the EC2 server
+    Type: String
+Resources:
+  ReplicationIoHaltedAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[Replication][${EnvironmentType}][SlaveIORunning] Not running"
+      AlarmActions: [!Ref OpsWarnSnsTopic]
+      InsufficientDataActions: [!Ref OpsWarnSnsTopic]
```

Comment:
We almost never use the short form array syntax in yaml (only because I almost never use it), so consistency would have this change to the other form. I prefer the long form because it diffs better.